lib/dealership.rb

```diff
@@ -0,0 +1,86 @@
+class Dealership
+  attr_reader :dealer_name,
+              :address,
+              :inventory,
+              :inventory_count,
+              :value
+
+  def initialize(dealer_name, address)
+    @dealer_name      = dealer_name
+    @address          = address
+    @inventory        = []
+    @inventory_count  = 0
+    @value            = 0
```

Comment:
I think `@value` is messing you up here. We don't need to set it as an instance variable and should only worry about it when calculating the total value.